src/BufferingParser.php

```diff
@@ -132,4 +128,23 @@ private function parseBody(string $body, string $boundary = null): Form
 
         return new Form($fields, $files);
     }
+
+    /**
+     * Parse the given content-type and returns the boundary if parsing is supported, false if it isn't or null if we are in url-encoded mode.
+     *
+     * @param string|null $contentType
+     *
+     * @return string|false|null
```

Comment:
Maybe `null|string` and empty string for `application/x-www-form-urlencoded`?

Reply:
by modifying the first part of parseBody ($boundary === null to $boundary === '') it could be done!